NS-Proto-Appendix.md

```diff
@@ -8,41 +8,27 @@ Converts shallowly an object to a record. If the object has a non-const value, a
 
 Checks whether the parameter is either a Record primitive or Record wrapper.
 
-## `Record.assign(...args: Record[]) -> Record`
-
-Merges all records passed as arguments into one **new** record returned by the function. The latest argument will override arguments before when merging.
-
-## `Record.entries(record: Record) -> Tuple`
-
-Returns all of the entries in a Record as a Tuple of 2-values Tuples (key, value) in the same order as `Record.keys()`.
-
 ## `Record.fromEntries(iterator: Iterator): Record`
 
-Takes an iterator of 2-element array-like values and creates a record out of it. If the iterator returns something other than an array-like tuple, a TypeError will be raised.
-
-## `Record.keys(record: Record) -> Tuple`
-
-Returns a Tuple containing the values serving as keys. Order will be the one defined in the main proposal document.
-
-## `Record.toString(record: Record) -> String`
-
-To be defined - it can actually show the full value.
+Similar to `Object.fromEntries`, but created a new `Record`.
 
-## `Record.values(record: Record) -> Tuple`
+## `Object` namespace functions working with `Record`
 
-Returns all of the values in a Record in the same order as the associated keys in `Record.keys()`.
+- `Object.is()`
+- `Object.entries()`
+- `Object.keys()`
+- `Object.values()`
 
-## `Object` namespace functions not available in `Record`
+## `Object` namespace functions not working with `Record`
 
+- `Object.assign()` - You can't assign new properties to a record.
```

Comment:
However, you should be able to pass records or Tuple as sources, no?

Reply:
Right, I will update the desc. The spec should already work, because it calls `ToObject` on the arguments.